packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -132,3 +134,35 @@ String? activeDevToolsServerAddress;
 
 /// The uri for the connected vm service protocol.
 String? connectedVmServiceUri;
+
+/// If memory allocation tracking is enabled, dispatch object creation.
+///
+/// This method is not member of FlutterMemoryAllocations, because
+/// [FlutterMemoryAllocations] should not increase size of the Flutter application
+/// if memory allocations are disabled.
+///
+/// Should be called only from within an assert.
+///
+/// Returns true to make it easier to be wrapped into `assert`.
+bool maybeDispatchObjectCreated(String library, String className, Object object) {
```

Comment:
Let's follow the naming convention for methods that should only be called from asserts and name this `debugMaybeDispatchObjectCreated`.